### PR TITLE
Got all the employees, and one employee, and pugged em both to the DOM

### PR DESCRIPTION
**Pull request by chrismellor08**
To test:

Go to localhost:4000/employees
Check to see all employees

AND 

go to localhost:400/employees/[id of your choice between 1-10]
Check to make sure there is data for one employee

### CODE REVIEW

**Comment by Rian501** [comment]
take out that console log, dude

File: controllers/employeeCtrl.js [hunk 1]
```diff
@@ -3,8 +3,27 @@
 module.exports.getEmployees = (req, res, next) => {
   const { Employee } = req.app.get('models');
   Employee.findAll() // love those built-in Sequelize methods
-  .then( (employees) => {
-    res.render('index', {employees});
+    .then( (employees) => {
+      let emps = employees.map( (emps) => {
+        return emps.dataValues;
+      });
+      console.log(emps);
```
Comment (Rian501) [inline controllers/employeeCtrl.js]: right chea

**Review by Rian501 — CHANGES_REQUESTED** [review]

**Comment by chrismellor08** [comment]
let console.logs = deleted.

**Comment by Rian501** [comment]
Works as advertised.

**Review by Rian501 — APPROVED** [review]

**Pull request merged**